staging/src/k8s.io/api/extensions/v1beta1/generated.proto

```diff
@@ -865,6 +871,11 @@ message PodSecurityPolicySpec {
   // +optional
   repeated AllowedFlexVolume allowedFlexVolumes = 18;
 
+  // AllowedCSIDrivers is a whitelist of inline CSI drivers that must be explicitly set to be embedded within a pod spec.
+  // An empty value means no CSI drivers can run inline within a pod spec.
+  // +optional
+  repeated AllowedCSIDriver allowedCSIDrivers = 23;
```

Comment:
Why is this placed here instead of at the end of the PodSecurityPolicySpec ?

Also, ordinal 22 is unoccupied.